app/services/bus_logic/restricted_bus.py

```
@@ -0,0 +1,947 @@
+        self.session.headers.update({
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36',
+            'Accept': 'application/json, text/javascript, */*; q=0.01',
+            'X-Requested-With': 'XMLHttpRequest',
+            'Referer': 'https://topis.seoul.go.kr/notice/openNoticeList.do'
+        })
+        
+        # Gemini 설정
+        if not gemini_api_key:
+            gemini_api_key = os.environ.get("GOOGLE_API_KEY") or os.environ.get("GEMINI_API_KEY")
+        if not gemini_api_key:
+            raise RuntimeError("Gemini API Key가 필요합니다. 환경변수 설정을 확인하세요.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)